### PR TITLE
22. Uploading a file

### PR DESCRIPTION
**Pull request by veronicatofino**

### CODE REVIEW

File: src/page/i-frame.page.ts [hunk 1]
```diff
@@ -2,9 +2,11 @@ import { $, browser, ElementFinder, promise } from 'protractor';
 
 export class IFramePage {
   private iFrame1: ElementFinder;
+  private title: ElementFinder;
 
   constructor() {
     this.iFrame1 = $('#frame1');
+    this.title = $('');
```
Comment (yesid-lopez) [inline src/page/i-frame.page.ts]: why is this selector empty?

File: src/page/personal-information.page.ts [hunk 2]
```diff
@@ -61,13 +66,30 @@ export class PersonalInformationPage {
     });
   }
 
+  private async uploadFile(fileToUpload: string) {
+    const absolutePath = this.path.resolve(__dirname, fileToUpload);
+    if (fs.existsSync(absolutePath)) {
+      await this.profilePictureField.sendKeys(absolutePath);
+    }else {
+      console.log('PATH NOT FOUND');
```
Comment (yesid-lopez) [inline src/page/personal-information.page.ts]: this console should not be there

File: src/page/personal-information.page.ts [hunk 3]
```diff
@@ -61,13 +66,30 @@ export class PersonalInformationPage {
...
+    }
+
```
Comment (yesid-lopez) [inline src/page/personal-information.page.ts]: delete empty line

File: src/page/i-frame.page.ts (hunk 1 above)
Comment (veronicatofino) [reply src/page/i-frame.page.ts]: That was from the previous point. I started to do it but for the change of URL that point couldn't be completed.

Comment (veronicatofino) [reply src/page/i-frame.page.ts]: Should I deleted it?

**Review by yesid-lopez — APPROVED** [review]

**Pull request merged**